addOns/httpsInfo/httpsInfo.gradle.kts

```diff
@@ -13,7 +13,7 @@ zapAddOn {
 
 dependencies {
     implementation("com.github.spoofzu:DeepViolet:5.1.16")
-    implementation("org.slf4j:slf4j-log4j12:1.7.6") {
+    implementation("org.slf4j:slf4j-log4j12:1.7.30") {
```

Comment:
Should use the bridge for Log4j 2.

Reply:
Same/similar to OpenAPI?
```

    implementation("org.apache.logging.log4j:log4j-slf4j-impl:2.14.0") {
        // Provided by ZAP.
        exclude(group = "org.apache.logging.log4j")
    }

    testImplementation("org.apache.logging.log4j:log4j-core:2.14.0")
```

Reply:
Yes, but the `testImplementation` should not be needed.

Reply:
Thanks

Reply:
No problem.